frontend/apps/picasso/defi/polkadot/pallets/Oracle.ts

```diff
@@ -4,26 +4,33 @@ import { fromChainIdUnit } from "shared";
 import { useStore } from "@/stores/root";
 import { CurrencyId } from "defi-interfaces";
 
-export async function subscribeAssetPrice(assetId: CurrencyId, api: ApiPromise) {
+export async function subscribeAssetPrice(
+  assetId: CurrencyId,
+  api: ApiPromise
+) {
   try {
-    const unsub: any = await api.query.oracle.prices(assetId.toString(), (prices: any) => {
-      const price = prices.isNone ? new BigNumber(0) : fromChainIdUnit(new BigNumber(prices.price.toString()));
-      useStore.setState({
-        oracle: {
-          prices: {
-            [+assetId.toString()]: {
-              price,
-              block: new BigNumber(prices.block.toString())
-            }
-          }
-        }
-      });
+    const unsub: any = await api.query.oracle.prices(
```

Comment:
`const assetIdStr = assetId.toString()`

and then use `assetIdStr`

Reply:
It is an unused assignment IMO, we don't use it anywhere else. and I guess 
```
assetIdStr == assetId.toString()
``` 
right?

Reply:
i wanted to eliminate the use of  `[+assetId.toString()]` with `assetIdStr`